packages/dart/lib/src/base/parse_constants.dart

```diff
@@ -1,7 +1,7 @@
 part of flutter_parse_sdk;
 
 // Library
-const String keySdkVersion = '3.1.0';
+const String keySdkVersion = '3.2.0';
```

Comment:
It seems these are all bug fixes, so the new version would be 3.1.2

Reply:
Yes. But we have package dependency updates. This can cause application conflicts.
I always increment the MINOR in these cases.